lib/cli/kit/config.rb

```diff
@@ -23,13 +23,13 @@ def initialize(tool_name:)
       # #### Example Usage
       # `config.get('name.of.config')`
       #
-      def get(section, name)
-        all_configs.dig("[#{section}]", name) || false
+      def get(section, name, default: false)
+        all_configs.dig("[#{section}]", name) || default
```

Comment:
`all_configs`... will that return a `false` by chance? If so, this would go to `default` automatically

Reply:
Nope - it's always strings